core/src/main/resources/hudson/model/Slave/help-remoteFS_fr.html

```diff
@@ -2,9 +2,9 @@
   <p>
   Un agent a besoin d'un répertoire dédié à Jenkins. Spécifiez ici le
   chemin absolu vers le répertoire de travail sur l'agent, par exemple
-  '/var/jenkins' ou 'c:\jenkins'. Cela doit être un chemin local à la machine
+  '/var/jenkins' ou 'c:\jenkins'. Le chemin doit être local à la machine
   agent. En règle générale, il n'est pas nécessaire que ce chemin soit
```

Comment:
```suggestion
  '/var/jenkins' ou 'c:\jenkins'. En règle générale, il n'est pas nécessaire que ce chemin soit
```

Reply:
WDYT about splitting with the dot (e.g. one sentence per line) to make the diff easier to read?

Reply:
we are going a bit far away from terminology changes, but why not.

Reply:
I mark this as resolved, I will open another PR to improve the French text overall.